### DISCUSSION TITLE
Adding Samba share to homepage

### DISCUSSION THREAD

**Discussion by Bert-Jan53** [IceWhaleTech/CasaOS#1760] [Q&A]
Probably just missing something, but;
I've installed CasaOS on an Ubuntu 22.04 server VM. I've added a Samba share to that server; the Media folder of my NAS. I've also added that mount to /etc/fstab. Works fine, I have access to the folder and read/write permissions. I can now access it via 'Files', by going to root - mnt - Media
I would like tot add that share to the CasaOS homepage under 'Storage'. At the moment I only see the main HDD there. I cannot figure out how to do that.|

I've also tried adding the samba share to 'Files' using smb://192.168.1.29/Media. That kinda works; I have access to the whole NAS, not just the Media folder. But I cannot get that to show up on the homepage either.

Is there any way to add shares to the homepage?

**Comment by ronnaudao** [comment]
Hey, did you get how? Im having the same trouble, i want to usa my NAS in plex, but i cant access. I only access by "files" using smb command
